beta-src/src/state/interfaces/GameStatusResponse.ts

```diff
@@ -0,0 +1,18 @@
+interface GameStatusResponse {
+  gameID: number;
+  countryID: number;
+  variantID: number;
+  potType: string;
+  turn: number;
+  phase: string;
+  gameOver: string;
+  pressType: string;
```

Comment:
Should this end up being an enum of the possible strings?

Reply:
maybe...i know that phase might also be an enum. at this stage though, we are getting these back as strings from the api...we can always change this later if we decide it's necessary?